pkg/kopia/cli/repository/storage/s3/s3_test.go

```diff
@@ -30,12 +30,12 @@ import (
 
 func TestNewS3(t *testing.T) { check.TestingT(t) }
 
-func newLocation(prefix, endpoint, region, bucket string, skipSSLVerify bool) internal.Location {
+func newLocation(prefix, endpoint, region string, skipSSLVerify bool) internal.Location {
 	return internal.Location{
 		"prefix":        []byte(prefix),
 		"endpoint":      []byte(endpoint),
 		"region":        []byte(region),
-		"bucket":        []byte(bucket),
+		"bucket":        []byte("bucket"),
```

Comment:
do we really need this change? what would happen if someone want to use another bucket?

Reply:
Make sense. The reason i hardcorded the value ist he bucket parameter of the `newLocation` function is always receiving the string "bucket" which is flagged as an issue by the `unparam` linter. Now, i fix it . 
Coud you please take look . 

Thank you